src/TLuaInterpreter.cpp

```diff
@@ -13769,3 +13770,21 @@ int TLuaInterpreter::startPermSubstringTrigger(const QString & name, const QStri
 
 }
 
+int TLuaInterpreter::alert( lua_State * L )
+{
+    int luaDurationMs;
+    if( ! lua_isnumber( L, 1 ) )
+    {
+        lua_pushstring( L, "alert: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaDurationMs = lua_tointeger( L, 1 );
+    }
+
+    QApplication::alert(mudlet::self(), luaDurationMs);
+
+    return 1;
```

Comment:
🐞  Should be `return 0;` as there is nothing that has been pushed onto the Lua stack to return here.